src/services/scriptingservice.h

```diff
@@ -175,7 +175,10 @@ class ScriptingService : public QObject {
     Q_INVOKABLE QList<int> selectedNotesIds() const;
 
     Q_INVOKABLE bool writeToFile(const QString &filePath,
-                                 const QString &data) const;
+                                 const QString &data,
```

Comment:
Can you please also add documentation for for the new parameters and methods in https://github.com/pbek/QOwnNotes/blob/develop/docs/content/scripting/methods-and-objects.md.